README.md

```diff
@@ -90,3 +90,45 @@ The extraction tool is [`i18next-parser`](https://www.npmjs.com/package/i18next-
 ).
 
 To workaround this, specify static values in `i18n.ts` file under any top-level directory below `src/app`. For example, `src/app/Settings/i18n.ts`.
+
+## ADDING QUICKSTARTS
+
+To add a new quickstart, create a new tsx/ts file under `src/app/QuickStarts` with your Quick Start name, like `my-quickstart.tsx`. 
+
+Cryostat's Quick Starts use a markdown extension which allows components to be highlighted using a button within the markdown in the Quick Start content itself. It was taken from OpenShift Console's GitHub repo and modified to fit Cryostat's needs.
+
+The following are taken from patternfly/patternfly-quickstarts GitHub repo.
```

Comment:
We probably dont need to mention this. Or maybe add something like `See More` if the content has more details on the PF Quickstart docs.

Reply:
And maybe remove this: `It was taken from OpenShift Console's GitHub repo and modified to fit Cryostat's needs.`